server/frontend/src/components/Bugs/Comments/PublicationForm.vue

```diff
@@ -205,6 +205,7 @@
 
 <script>
 var Mustache = require("mustache");
+import { Base64 } from "js-base64"; // eslint-disable-line import/no-unresolved
```

Comment:
Any idea why eslint can't detect this in package.json?

Reply:
My bad: eslint was a bit mis-configured, the `import/resolver` was overriding the webpack path (mostly because I do that on another project). The issue was that eslint did not find anymore the correct paths towards node-modules, from sub-components.

It's now fixed in 7586f1ce8b7eb330909c9e70f7671f1b31a9f48c : you can check by editing the `js-base64` import to an invalid one (say `js-basewhatever`), and it will then fail.